README.md

```diff
@@ -85,6 +85,16 @@ Run Touchégg manually by running the command `touchegg` or reboot to get starte
 
 ## Red Hat, Fedora and derivatives
 
+On Fedora, it is recommended to use the
```

Comment:
I created the "official" repo here:
https://copr.fedorainfracloud.org/coprs/jose_exposito/touchegg/

It is still not working, but once your pull request is merged, it should work.
Do yo mind changing the URL, please?

Reply:
Done!

Reply:
I made a quick test with other distros and I think it's safe to add them to the list:

```suggestion
On CentOS, Fedora, Mageia, OpenMandriva and openSUSE it is recommended to use the
```

Reply:
I made a [small change](https://github.com/JoseExposito/touchegg/commit/acc73a0268715892a2ebad7888a7272ea353e3ef) to support openSUSE and I think we can add it to the list. We can also support CentOS 8 (EPEL) :rocket:  

```suggestion
On Fedora, openSUSE and CentOS (EPEL) it is recommended to use the
```